engine/src/flutter/shell/platform/darwin/ios/framework/Headers/FlutterPlugin.h

```diff
@@ -44,23 +44,40 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  * Called if this has been registered for `UIApplicationDelegate` callbacks.
+ *
+ * This method is deprecated and will not be called when this plugin conforms to
+ * `FlutterSceneLifeCycleDelegate`.
  */
-- (void)applicationDidBecomeActive:(UIApplication*)application;
+- (void)applicationDidBecomeActive:(UIApplication*)application
+    API_DEPRECATED("Implement ``FlutterSceneLifeCycleDelegate/sceneDidBecomeActive:`` instead.",
```

Comment:
The syntax came from https://github.com/alexey-lysiuk/macos-sdk/blob/c47669c5dac76e2f15970587c5a7274c09631aa2/MacOSX15.5.sdk/System/Library/Frameworks/GameKit.framework/Versions/A/Headers/GKScore.h#L12

The `/` delimiter does seem a bit weird but there doesn't seem to be a convention.